documentation/en/source/pages/templates-spec/builders/builders-ovf.rst

```diff
@@ -61,7 +61,7 @@ If you are using YAML:
 	- type: OVF or OVA
 	  hardwareSettings:
 	    memory: 1024
-	    hwType: 7
+	    hwType: 7 (ESXi>4.x)
```

Comment:
This is not a problem of documentation, but I don't understand why we ask this the customer. This is not user friendly. I will do some research on my side